class-prep/9/exercises.md

```diff
@@ -5,9 +5,9 @@ currentMenu: classes
 
 1. Let's use a ViewModel in our `Edit` action methods. If you don't have edit functionality already, you can add it by following the steps in [class 8 prep exercises](../8/exercises.html). Change the name of this class to `AddEditCheeseViewModel` to more properly reflect its use.
```

Comment:
Is the new ViewModel supposed to be called `AddEditCheeseViewModel`, or `EditCheeseViewModel`? Wasn't sure if this was a typo or not.